tests/ImageSharp.Tests/IO/ChunkedMemoryStreamTests.cs

```diff
@@ -167,18 +167,20 @@ public void MemoryStream_WriteToTests()
     [Fact]
     public void MemoryStream_WriteToSpanTests()
```

Comment:
Write tests could stress more cases with different sizes. See my comment on the read tests.